### PR TITLE
Update docs to handle all settlement currencies

### PR DESCRIPTION
**Pull request by piedup**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -529,50 +529,37 @@ mutation PrepareBid($input: prepareBidInput!) {
...
+        ...AuthorizationRequestFragment
       }
     }
   }
 }
+${authorizationRequestFragment}
 ```
 
-1. Sign all `AuthorizationRequest` objects and build the `bidInput` argument.
+`AuthorizationRequestFragment` is defined in [authorizations.js](authorizations.js).
```
Comment (redox) [inline README.md:540]:
```suggestion
`AuthorizationRequestFragment` is defined in [authorizations.js](./authorizations.js).
```

File: README.md [hunk 2]
```diff
@@ -529,50 +529,37 @@ mutation PrepareBid($input: prepareBidInput!) {
...
+
+3. Sign all `AuthorizationRequest` objects and build the `bidInput` argument. `buildApprovals` is defined in [authorizations.js](authorizations.js).
```
Comment (redox) [inline README.md:542]:
```suggestion
3. Sign all `AuthorizationRequest` objects and build the `bidInput` argument. `buildApprovals` is defined in [authorizations.js](./authorizations.js).
```

File: yarn.lock [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (utay) [inline yarn.lock:1]: To remove

File: node_modules/.yarn-integrity [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (utay) [inline node_modules/.yarn-integrity:1]: To remove

**Review by redox — APPROVED** [review]
We need the links to `authorization.js` to include `./` to work as far as I remember

File: authorizations.js [hunk 5]
```diff
@@ -0,0 +1,96 @@
+const { gql } = require('graphql-request');
```
Comment (utay) [inline authorizations.js:1]: To move into examples?

File: README.md (hunk 1 above)
Comment (utay) [reply README.md:540]: Any reason to have it root level and not into examples?

Comment (piedup) [reply README.md:540]: It was in examples first, but I felt like it's more of a a helper code than an example. Can move it there if you prefer. Let me know.

Comment (utay) [reply README.md:540]: I see, I'd be inclined to move it into examples as the js project is setup there with the other files. We can reference `./examples/authorizations.js` from the README

Comment (piedup) [reply README.md:540]: ok

**Pull request merged**